contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaTopicOffsetManager.java

```diff
@@ -0,0 +1,487 @@
+/**
```

Comment:
Looks like this code depends on some internal API, can we verify if it also support 0.8.1.1?